gui/templates/system/manualupdate_wizard_1.html

```diff
@@ -19,8 +19,11 @@
 {% block form %}
 <tr>
   <td colspan="2">
-    {% url "system_configsave" as saveurl %}
-    <p>{% blocktrans with url=saveurl %}Consider downloading your configuration before proceeding, <a href="{{ url }}" target="_blank">click here</a>.{% endblocktrans %}</p>
+    <p>{% trans "Consider downloading your configuration before proceeding" %}
```

Comment:
How does that look visually, with "click here" right after the text, without a comma?